include/sendemail.php

```diff
@@ -495,18 +495,17 @@ function get_email_summary($buildid, $errors, $errorkey, $maxitems, $maxchars, $
             $information .= "\n";
 
             foreach ($tests as $test) {
-                $info = $test['name'] . ' (' . $serverURI . '/testDetails.php?test=' . $test['id'] . '&build=' . $buildid . ")\n";
+                $info = "{$test['name']} {$test['details']} ({$serverURI}/testDetails.php?test={$test['id']}&build={$buildid}\n";
                 $information .= substr($info, 0, $maxchars);
             }
             $information .= "\n";
             return $information;
         };
 
-        $information .= $AddTestsToEmail($build->GetFailedTests($maxitems), 'Tests failing');
+        $information .= $AddTestsToEmail($build->GetFailedTests($maxitems), 'Tests not passing');
```

Comment:
This section should still be called "Tests failing", as "Tests not run" appear in a separate section later on.